bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/PWA.java

```diff
@@ -0,0 +1,94 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.cq.wcm.core.components.models;
+
+/**
+ * Interface for reading progressive web apps configuration
+ */
+public interface PWA {
+
+    /**
+     * Property that indicates whether the page is PWA enabled
+     * @since com.adobe.cq.wcm.core.components.models 12.18.0
+     */
+    final String PROP_PWA_PWAENABLED = "pwaEnabled";
```

Comment:
We generally use `PN_*` pattern for property names, see https://github.com/adobe/aem-core-wcm-components/blob/feature/pwa/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/PWA.java#L31-L45